server/routes/stock-details.js

```diff
@@ -0,0 +1,56 @@
+var express = require('express');
+var router = express.Router();
+
+const defaultDetails = {
+    company_name: "",
+    symbol: "",
+    previous_close: 0,
+    open: 0,
+    last_price: 0,
+    change: 0,
+    change_percent: "",
+    currency: "",
+    volume: "",
+    shares_owned: 0,
+    avg_vol: "",
```

Comment:
I believe this should be an int, not a string, so default val should = 0
same for market_cap below